.env

```diff
@@ -17,5 +17,7 @@ MAPBOX_STYLE_URL='mapbox://styles/covid-nasa/ckb01h6f10bn81iqg98ne0i2y'
 # Google form for feedback
 GOOGLE_FORM = 'https://docs.google.com/forms/d/e/1FAIpQLSfGcd3FDsM3kQIOVKjzdPn4f88hX8RZ4Qef7qBsTtDqxjTSkg/viewform?embedded=true'
 
-# Google analytics tracking code
-GOOGLE_ANALYTICS_ID='G-CQ3WLED121'
+# Google Tag Manager tracking code
+GOOGLE_TAG_MANAGER_ID='GTM-KHK5WJQ'
+GOOGLE_TAG_AUTH='tqd01PxWYuaeIneNeoNm1g'
+GOOGLE_TAG_PREVIEW='env-1'
```

Comment:
Aren't these supposed to be secrets, in which case shouldn't we have them in `env.local` ?

Reply:
Hey yeah so I am not sure tag manager id is sensitive or not, but it is a better idea to move them when I am not sure! so I moved them to Netlify, and github action. I took this as an opportunity to set up a different key for dev environment.